Example/PrimerSDK/CreateClientToken.swift

```diff
@@ -11,6 +11,48 @@ import PrimerSDK
 
 enum Environment: String, Codable {
     case local, dev, sandbox, staging, production
+    
+    init(intValue: Int) {
+        switch intValue {
+        case 0:
+            self = .local
+        case 1:
+            self = .dev
+        case 2:
+            self = .sandbox
+        case 3:
+            self = .staging
+        case 4:
+            self = .production
+        default:
+            fatalError()
+        }
+    }
+    
+    var intValue: Int {
+        switch self {
+        case .local:
+            return 0
+        case .dev:
+            return 1
+        case .sandbox:
+            return 2
+        case .staging:
+            return 3
+        case .production:
+            return 4
+        }
+    }
+    
```

Comment:
I was wondering whether the value of _simplifying_ this enum a bit more.
I wouldn't do that if we planned to handle the `default` value with something different.
But since we are sending a `fatalError()` (that should never happen in any case) I thought that we could benefit from swift a bit more.
In this case, whenever we add or remove a `case` we will have less code to deal with.

If you don't see the value in it, I'm fine with the current implementation as this is the sample app.

```suggestion
enum Environment: String, Codable, CaseIterable {
    case local, dev, sandbox, staging, production
    
    init(intValue: Int) {
        self = Environment.allCases[intValue]
    }
    
    var intValue: Int {
        Environment.allCases.firstIndex(of: self) ?? 0
    }    
```